File: app/src/main/java/me/ykrank/s1next/data/api/model/wrapper/BaseDataWrapper.java
package me.ykrank.s1next.data.api.model.wrapper;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

import me.ykrank.s1next.data.api.model.Account;

@SuppressWarnings("UnusedDeclaration")
@JsonIgnoreProperties(ignoreUnknown = true)
public class BaseDataWrapper<T extends Account> extends OriginWrapper<T> {

}

File: app/src/main/java/me/ykrank/s1next/data/api/model/wrapper/BaseDataWrapper.kt
package me.ykrank.s1next.data.api.model.wrapper

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import me.ykrank.s1next.data.api.model.Account

@JsonIgnoreProperties(ignoreUnknown = true)
open class BaseDataWrapper<T : Account> : OriginWrapper<T>()
